services/frontend-service/src/ui/ReleaseDialog.tsx

```diff
@@ -91,26 +91,31 @@ const VersionDiff = (props: { current: number | undefined; target: number; relea
     }
 };
 
-const QueueDiff = (props: { queued: number; current: number; releases: Release[] }) => {
+// target is the version we are looking at currently:
+const QueueDiff = (props: { queued: number; target: number; current: number; releases: Release[] }) => {
```

Comment:
I think we can remove the `current` prop from here?

Reply:
done